uperf/uperf_run

```diff
@@ -784,8 +785,8 @@ else
 		save_pbench_results
 	fi
 fi
-for client in $client_ip_list; do
-	ssh -oStrictHostKeyChecking=no root@$client "pkill -9 uperf"
+for client in $server_ip_list; do
+	ssh -oStrictHostKeyChecking=no root@$client "pkill -x -9 uperf"
```

Comment:
Can you add a comment about why -X is needed so nobody thinks they can be economical in the future and delete it?